spark/src/main/kotlin/com/adevinta/spark/components/chips/FilterChip.kt

```diff
@@ -105,6 +90,10 @@ import androidx.compose.material3.FilterChip as MaterialFilterChip
  * for this chip. You can create and pass in your own `remember`ed instance to observe
  * [Interaction]s and customize the appearance / behavior of this chip in different states.
  */
+@Deprecated(
+    "Use one of the options: ChipOutlined, ChipFilled, ChipTinted, ChipDashed",
```

Comment:
Material Filter Chip is closer to TintedChip than the other styles variant so we should recommend this one only instead